### PR TITLE
Improve Events part of the documentation

### PR DESCRIPTION
**Pull request by simplykatsa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,11 +301,35 @@ request_headers: '{
 
 ## Events
 
-Events are a way to organize code. They allow you to add your logic to existing commands. They are executed asynchronously in the background.
+**Events** in platformOS allow you to record that something has happened in the system. Think of them as logs that capture these actions. When an event occurs, other parts of the application might need to react to it. These events are stored using GraphQL mutations and can include various arguments detailing the specifics of what happened. They enable you to add your logic to existing commands and are executed asynchronously in the background.
```
Comment (Slashek) [inline README.md:304]:
```suggestion
**Events**allow you to record that something has happened in the system. Think of them as logs that capture these actions. When an event occurs, other parts of the application might need to react to it. In platformOS, these events are stored using GraphQL mutations and can include various arguments detailing what happened. They enable you to add logic to existing commands and are executed asynchronously in the background.
```

Events are generic concept, so move "in platformOS" when we discuss platformOS specific thing, which is events implementation :)

File: README.md [hunk 2]
```diff
@@ -301,11 +301,35 @@ request_headers: '{
...
+
+### Example Scenario
+
+Imagine you have an e-commerce platform where users can place orders. When an order is placed, several things need to happen:
+1. The order needs to be saved in the database.
+2. An email confirmation needs to be sent to the user.
+3. The inventory needs to be updated.
+4. The sales team needs to be notified.
+
+By creating an event when the order is placed, each of these actions can be handled separately by different parts of the application, ensuring a clean and manageable codebase.
```
Comment (Slashek) [inline README.md:320]: 👍

File: README.md [hunk 3]
```diff
@@ -301,11 +301,35 @@ request_headers: '{
...
+
+Here's how you can record an event in platformOS:
```
Comment (Slashek) [inline README.md:322]: We might want to add a cross-reference to https://documentation.platformos.com/developer-guide/activity-feeds/activity-feeds - those are not regular records, we do not use `record_create`, but `activity_create`

**Review by Slashek — APPROVED** [review]
Awesome improvements, thanks! Please take a look at the comments before merging, but even in the current form, it's a great improvement!

File: README.md (hunk 3 above)
Comment (simplykatsa) [reply README.md:322]: Great idea! Added it.

**Pull request merged**